package.json

```
@@ -0,0 +1,69 @@
+{
+  "name": "Team3App",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "android": "react-native run-android",
+    "ios": "react-native run-ios",
+    "start": "react-native start",
+    "test": "jest",
+    "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
+    "postinstall": "husky install"
+  },
+  "dependencies": {
+    "@types/react-native-dotenv": "^0.2.0",
```

```suggestion
devDependencies: {
    ...
    "@types/react-native-dotenv": "^0.2.0",
    ...
}
```

개발 의존성으로 옮겨도 될 거 같아요!